test/test_debugger.cpp

```diff
@@ -632,8 +632,14 @@ bool debugger_client::test_inspect_variables()
     m_client.send_on_control("debug_request", make_inspect_variables_request(0));
     nl::json rep = m_client.receive_on_control();
 
-    nl::json var = rep["content"]["body"]["variables"];
-    bool res = var["i"] == 4 && var["j"] == 8 && var["k"] == 5;
+    nl::json vars = rep["content"]["body"]["variables"];
+    std::cout << vars << std::endl;
+    std::cout << vars.size() << std::endl;
+    size_t size = vars.size();
+    bool res = size >= 3;
+    res = res && vars[size-3]["name"] == "i" && vars[size-3]["value"] == 4;
```

Comment:
This should be improved by searching for the variable in the array of variables, instead of relying on a specific order.